CHANGELOG.md

```diff
@@ -6,6 +6,13 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## 1.1.0
+
+This release includes:
+
+* [pull request #49: Update bundled Sass config to `stylelint-config-standard-scss@11`](https://github.com/alphagov/stylelint-config-gds/pull/49)
+* [pull request #53: Update Stylelint minimum version to `stylelint@15.11.0`](https://github.com/alphagov/stylelint-config-gds/pull/53)
```

Comment:
**question** What's the impact of the change of `peerDependencies` for teams using `stylelint-config-gds`? Is that something that `npm` will pick up automatically when updating `stylelint-config-gds` or will people have to do it themselves? Just a bit worried about the impact down the line, but hoping `npm` has us covered there (which would explain your choice of non-breaking version increase) 😊

Reply:
@romaricpascal Yeah it's all automatic since `npm@7` so those `peerDependencies` don’t need installing or updating separately

Did you follow the PRs to see that the major bump was already included by another package?

* https://github.com/alphagov/stylelint-config-gds/pull/49#pullrequestreview-1628709938

Reply:
@romaricpascal It's had another bump that supersedes this one so I've rebased and amended:

* https://github.com/alphagov/stylelint-config-gds/pull/57